doc-content/drools-docs/src/main/asciidoc/PMML/pmml-support-con.adoc

```diff
@@ -34,3 +35,7 @@ endif::[]
 ifdef::DROOLS,JBPM,OP[]
 <<_builddeployutilizeandrunsection>>.
 endif::[]
+
+ifdef::DM,PAM[]
+You can migrate a PMML service to a {KOGITO} microservice. For more information about migrating to {KOGITO} microservices, see {URL_GETTING_STARTED_KOGITO_MICROSERVICES}#assembly-getting-started-migration-to-kogito-microservices[_{MIGRATION_KOGITO_SERVICES}_].
+endif::[]
```

Comment:
Link is not rendering correctly.

![image](https://user-images.githubusercontent.com/3627775/136169521-e4de921b-dc9b-407b-a4f5-fa5a4e52f983.png)

Reply:
Yes, I am aware of this link. Once Heena merges her PR I am going to rebase so that it will be reflected automatically in the doc. Thank you so much for pointing that out. :)